ibm_mq/tests/test_ibm_mq.py

```diff
@@ -86,6 +90,32 @@ def test_check(aggregator, instance, seed_data):
     aggregator.assert_service_check('ibm_mq.channel', check.CRITICAL, tags=bad_channel_tags)
 
 
+@pytest.mark.usefixtures("dd_environment")
+def test_service_check_from_status(aggregator, instance, seed_data):
+    check = IbmMqCheck('ibm_mq', {}, {})
+
+    service_check_map = {
```

Comment:
Looks like a duplicate of `SERVICE_CHECK_MAP` in `ibm_mq.py`.
Can we share a single constant?

Reply:
Actually I did that on purpose. A way to decouple the test from the code . If the mapping in the code changes, you have to change the test to make it pass again. However, if you changed the mapping by error, you will notice it thanks to the test.

If we use the same mapping, the test will never fail if someone modify the mapping by error or delete one of the mapping entry by error.